### ISSUE TITLE
Merge this repo back into pelias/admin-lookup

### ISSUE THREAD

**Issue by orangejulius** [pelias/wof-admin-lookup#48]
It feels rather silly to have two admin lookup repos, especially when one of them is deprecated. I propose we move all the code in this one into the pelias/admin-lookup repo, and then delete this one. We may change how we do admin lookup again in the future, so locking Who's on First into the title of the repository doesn't feel right.

**Comment by trescube** [comment]
Do it, I introduced the wof-admin-lookup project in my younger more inexperienced JS days.

**Comment by orangejulius** [comment]
That's true, I like that as well. We do kinda have that with https://github.com/pelias/wof-pip-service/ though.

**Comment by dianashk** [comment]
Just to clarify, because I think my previous statement may have been a bit vague... An importer would do something like this:

```javascript
var admin_lookup = require('pelias-admin-lookup');
var config = { admin_lookup: { source: 'wof' } };

someStreams
  .pipe(admin_lookup.setup(config.admin_lookup)
  .pipe(.....);
```

and in the `admin-lookup` code it would just create the right one based on the config.

**Comment by orangejulius** [comment]
Sorry, my comment was also vague! :p

I really like the interface you suggested (and was also thinking of something along those lines when reading your comment, and was in fact agreeing. Right now we have `wof-pip-service`, which has a lot of, but not all of, the logic for doing admin lookup with WOF. Maybe that could eventually become `wof-admin-lookup`, and this repo could be `admin-lookup`
